Gemfile

```diff
@@ -70,10 +72,6 @@ end
 group :test do
   gem 'capybara'
   gem 'database_cleaner'
-  gem 'factory_girl_rails'
-  gem 'rspec'
-  gem 'rspec-rails'
-  gem 'selenium-webdriver'
   gem 'webmock'
```

Comment:
I deleted these because they were duplicates (we already have them in the gemfile & bundler kept complaining about them everytime I did `bundle install`